lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/feature/Feature.json

```diff
@@ -33,6 +33,10 @@
 		{
 			"name": "ALLOW_AGENT_WRITES",
 			"description": "When enabled, bedrock agents are allowed to directly write to Synapse."
+		},
+		{
+			"name": "REQUIRE_TWO_FA_BYPASS",
+			"description": "When enabled, two factor authentication WON'T be required for users accessing Synapse."

```

Comment:
DISABLE_2FA_REQUIREMENT
Set this value to "false" to require 2FA for all users.